index-test.js

```diff
@@ -1,20 +1,59 @@
-/*
+// MINIMUM requirement of Unit Tests (as stated on project brief)
 
-MINIMUM requirement of Unit Tests (as stated on project brief)
+//test("Submitting a new task adds it to the list", () => {
+// test goes here
+//});
 
-test("Submitting a new task adds it to the list", () => {
-  // test goes here
-});
+setTimeout(() => {
```

Comment:
Great idea on using the setTimeout so that the tests pick up the selected element from the input, because there  created elements in JavaScript to give it the time to create a list, great work on this 🙌 💯

Reply:
This was good thinking, Maria!